scripts/abi_artifact_generator/config/architectures.yml

```diff
@@ -73,6 +79,8 @@
       bl {{ function_name }}
     return_from_function: |
       bx lr
+    return_from_function_with_cleanup: |
+      .word 0xe7f0def0
```

Comment:
What's this? A comment at least!

Reply:
I just googled around for arm's invalid instructions. And this is the most robust thing I found.

Reply:
Added the comment.